src/style.css

```diff
@@ -0,0 +1,93 @@
+body {
+  margin: 0;
+}
+
+.product-card {
+  margin: auto;
+  width: 200px;
+  height: 403px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 14px 15px;
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
```

Comment:
To best sizing try to use only width or height.

Reply:
Using one of this properties not possible to pass test. If i use both- it pass perfectly

Reply:
I found out the problem. With width test not pass, only with height. Why i dont know..